_projects/2023/stingray/lombscargle.md

```diff
@@ -0,0 +1,51 @@
+---
+name: Astronomy with Unevenly Sampled Data
+desc: Implementing time series analysis methods for unevenly sampled data for X-ray astronomy data 
+requirements:
+ - Python 
+ - Experience with time series methods and/or Fourier analysis welcome
+difficulty: medium
+issues:
+# Related issues (if any)  to this project.
+- https://github.com/StingraySoftware/stingray/issues/689
+mentors:
+# First person in contact; mentors may change before project starts.
+# GitHub handles
+ - dhuppenkothen
```

Comment:
not urgent for now, but it would be great to have someone listed as a second mentor